result/bgpv4.yaml

```diff
@@ -144,3 +146,27 @@ components:
             Number of Notification messages received.
           type: integer
           x-field-uid: 15
+        fsm_state:
+          description: >-
+            Bgp peer fsm state as idle, connect, active, opensent, openconfirm and established. In all the states except Established the bgp session is down,
+            Idle refers to the Idle state of the FSM.
+            Connect refers to the state where the session is waiting for the underlying transport session to be established.
+            Active refers to the state where the session is awaiting for a connection from the remote peer.
+            OpenSent refers to the state where the session is in the process of being established. The local system has sent an OPEN message.
+            OpenConfirm refers to the state where the session is in the process of being established. The local system is awaiting a NOTIFICATION or KEEPALIVE message from remote peer.
```

Comment:
local system has sent and received an OPEN message and is awaiting ...

Reply:
updated